scripts/update_stm32_package.py

```diff
@@ -9,7 +9,9 @@
 # usage : 'python update_stm32_package.py <stm32_serie>'
 
 import os
-import sys; sys.dont_write_bytecode = True    # do not generate pyc file
```

Comment:
Purpose of this commit is unclear

Reply:
@FRASTM, Can you tell me the rational behind this ?

Reply:
This is to prevent Python 2.x from generating `.pyc` files. It's better to just add these extension to gitignore, even though Python 2.x is now obsolete.